### PR TITLE
Added organic replicator

### PR DESCRIPTION
**Pull request by AndreySolodovnikov**
This machine can clone seeds using some UUMater and energy, the better are seeds, the more UUM is necessary

### CODE REVIEW

File: src/main/java/gregtech/api/util/GT_Recipe.java [hunk 1]
```diff
@@ -512,8 +512,8 @@ public GT_Recipe_AssemblyLine(ItemStack aResearchItem, int aResearchTime, ItemSt
...
-        public static final GT_Recipe_Map sAssemblylineVisualRecipes = new GT_Recipe_Map(new HashSet<GT_Recipe>(30), "gt.recipe.fakeAssemblylineProcess", "Assemblyline Process", null, RES_PATH_GUI + "FakeAssemblyline", 1, 1, 1, 0, 1, E, 1, E, true, false);
+        public static final GT_Recipe_Map sOrganicReplicatorFakeRecipes = new GT_Recipe_Map(new HashSet<GT_Recipe>(1),"gt.recipe.organicreplicaator","Organic Replicator", null,RES_PATH_GUI + "basicmachines/OrganicReplicator",1,1,1,1,1,E,1,E,true,true);
+        //public static final GT_Recipe_Map sAssemblylineFakeRecipes = new GT_Recipe_Map(new HashSet<GT_Recipe>(30), "gt.recipe.scanner", "Scanner", null, RES_PATH_GUI + "basicmachines/Default", 1, 1, 1, 0, 1, E, 1, E, true, true); public static final GT_Recipe_Map sAssemblylineVisualRecipes = new GT_Recipe_Map(new HashSet<GT_Recipe>(30), "gt.recipe.fakeAssemblylineProcess", "Assemblyline Process", null, RES_PATH_GUI + "FakeAssemblyline", 1, 1, 1, 0, 1, E, 1, E, true, false);
```
Comment (draknyte1) [inline src/main/java/gregtech/api/util/GT_Recipe.java:516]: Doesn’t need formatting changes.

**Review by draknyte1 — APPROVED** [review]
I honestly don’t see why this needs to be a unique machine. 
A simple Boolean, onScrewDriverRightClick() and save/load NBT is all you need to toggle mode & recipe map..

**Comment by Blood-Asp** [comment]
GT has special machines for everything. So i guess it fits.

**Comment by draknyte1** [comment]
Pointless use of Metas. 
Pointless more machines. 

Scanning is scanning. Which logically is why; It would be best to adapt the existing scanner.

**Comment by MauveCloud** [comment]
I'm confused about the usefulness of this PR.

As I understand it, a player can already get more crops of a given type by crossbreeding.  In what situation would it become advantageous to use UUM in a machine to clone the seeds instead?

**Comment by AndreySolodovnikov** [comment]
I’ve decided to make this machine after 6 hours of planting only ONE field of tungsten, it was too boring and long, gameplay shouldn’t be like that

**Comment by SapientMC** [comment]
@MauveCloud What is the maximum number of crops fields you had?
This machine are very useful. Its the only way to make industrial number of crops with best stats like 23-31-0.

**Comment by SapientMC** [comment]
@draknyte1 
> Scanning is scanning.

That's true, but replicating from UU-mater are not scanning, so it is a specialized hybrid of scanner and mass fabricator.

**Pull request merged**